src/Nordea/Components/TextInput.elm

```diff
@@ -43,6 +23,7 @@ type alias Config msg =
     { value : String
     , onInput : Maybe (String -> msg)
     , placeholder : Maybe String
+    , error : Maybe Bool
```

Comment:
Is the meaning clearer without a maybe? 🤔 In this case we don't have to question the difference between `Nothing` and `Just False`

```suggestion
    , showError : Bool
```

Reply:
Would this mean that every text input implementet now need to have withError False to it? 
Or will this mean that error/showError: Bool and that it is always default set to False in the model instead of Nothing?

Reply:
I think it makes sense to set it to False by default, and then use `withError True` in cases where it should be true

Reply:
I agree ^

Reply:
Agree! Changed it now 😊

Reply:
@fredrikloberg @fshauge should withError be renamed to showError as well, or should we keep it as is?

Reply:
I think both are fine